packages/nextjs/pages/index.tsx

```diff
@@ -6,19 +7,51 @@ const Home: NextPage = () => {
     <>
       <MetaHeader />
       <div className="flex items-center flex-col flex-grow pt-10">
-        <div className="px-5">
-          <h1 className="text-center mb-8">
+        <div className="px-5 w-[90%] md:w-[75%]">
```

Comment:
is this styles from another challenge?

Reply:
This based on this [Base File ](https://github.com/scaffold-eth/se-2-challenges/blob/challenge-0-simple-nft/packages/nextjs/pages/index.tsx)